app/components/UserProfile/UserProfile.tsx

```
@@ -74,25 +70,37 @@ const ProfilePage: NextPage = () => {
-        setMemberData(activeAccount); // Example: setting active account as member data
-        setBalance('0'); // Example: set balance to 0 or fetch actual balance
+        setOwnedIds(ownedTokenIds);
       }
     };
 
     fetchData();
-  }, [activeAccount, ownedIds, unlockContract]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [activeAccount]);
+
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: unlockContract

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)